app/src/app/routes/Sources/Sources.tsx

```diff
@@ -19,7 +22,11 @@ const Sources = (): JSX.Element => {
   return (
     <Container maxWidth="xl">
       <Grid className={classes.gridContainer} spacing={3} container>
-        <SourceCreate />
+        <SourceCreate
+          className={classes.button}
+          color="primary"
+          variant="contained"
```

Comment:
I don't think `SourceCreate` is meant to be generic, which would allow those props to stay inside of this component implementation don't you think ?

Of course, we could still spread `ButtonProps` inside, and this would allow style overwriting.